packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-10/upgradeHelpers.mjs

```diff
@@ -8,17 +8,12 @@ import {
   VALIDATORADDR,
   USER1ADDR,
   CHAINID,
-  HOME,
 } from '../constants.mjs';
 
-import { agd, agops } from '../cliHelper.mjs';
+import { agd } from '../cliHelper.mjs';
 
 export * from '../coreUpgradeHelpers.mjs';
```

Comment:
given that all the modules are in the same source tree and aren't exported, I think it would be more clear to import from where exports are defined. consider removing this `export *` so that when reading a test the developer can see what module defines it (and thereby whether it's special to this test or shared by others)